### PR TITLE
[zh] Sync setup: install-kubeadm.md

### PR DESCRIPTION
**Pull request by win5923**
Sync v1.28 setup/production-environment/tools/kubeadm/install-kubeadm.md

### CODE REVIEW

File: content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -319,55 +319,120 @@ You can create this directory if you need to, making it world-readable but write
 如有需要，你可以创建此目录，并将其设置为对所有人可读，但仅对管理员可写。
 {{< /note >}}
 
+### Google托管的软件包仓库（已弃用） {#dpkg-google-package-repo}
```
Comment (tengqm) [inline content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:322]: Thanks for the contribution.
However, please check the Chinese localization style guide and make sure you are following the guide. We usually keep the original English text in comments, in the translated markdowns. That is for ease of review and change tracking.

Comment (win5923) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:323]: finished! thx.

Comment (tengqm) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:323]: Please check how we commented out existing English texts, i.e. put `<!--` and `-->` on their own lines, preserve the existing line wrappings, etc.
All these conventions are to make comparison between the English text and the Chinese
translation easier.
Please do read the [Chinese localization style guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).

Comment (win5923) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:325]: Please check for any issues, thanks.

Comment (win5923) [reply content/zh-cn/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:325]: Sorry, I don't know how to change the email address for a previous commit.

**Pull request closed without merging**